src/vs/workbench/contrib/notebook/common/notebookEditorModel.ts

```diff
@@ -377,7 +377,7 @@ export class ComplexNotebookEditorModel extends EditorModel implements INotebook
 			if (!this.isResolved()) {
 				return;
 			}
-			const success = await this._contentProvider.save(this.notebook.uri, CancellationToken.None);
+			const success = false;
```

Comment:
Not familiar enough with this code to do a better fix

The interactive window always returns `false` for these methods